### PR TITLE
Add ruleset for pcshannover.de #15635

### PR DESCRIPTION
**Pull request by RolloKraemer**

### CODE REVIEW

File: src/chrome/content/rules/pcshannover.de.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Thinkwiki.org">
```
Comment (YegorIevlev) [inline src/chrome/content/rules/pcshannover.de.xml]: `<ruleset name="pcshannover.de">`

File: src/chrome/content/rules/pcshannover.de.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	<target host="pcshannover.de" />
+	<target host="www.pcshannover.de" />
+
```
Comment (YegorIevlev) [inline src/chrome/content/rules/pcshannover.de.xml]: `<securecookie host=".+" name=".+" />`

**Comment by SabrinaFinkel** [comment]
Any updates?

**Comment by Bisaloo** [comment]
@J0WI, I submitted a commit to move forward on this PR. Could you double check it's okay please?

**Pull request merged**